src/Neo/NeoSystem.cs

```diff
@@ -211,16 +208,6 @@ public void EnsureStopped(IActorRef actor)
             inbox.Receive(TimeSpan.FromMinutes(5));
         }
 
-        /// <summary>
-        /// Loads an <see cref="IStore"/> at the specified path.
-        /// </summary>
-        /// <param name="path">The path of the storage.</param>
-        /// <returns>The loaded <see cref="IStore"/>.</returns>
-        public IStore LoadStore(string path)
```

Comment:
This is public, maybe we should not just delete it like this.

Reply:
Is `Load` but it only returns the `StoreFactory` result (that is public too)

Reply:
mmm maybe it's used by plugis for create a new one, let me check